elixir/apps/domain/test/domain/billing_test.exs

```diff
@@ -44,6 +44,30 @@ defmodule Domain.BillingTest do
     end
   end
 
+  describe "users_limit_exceeded?/2" do
+    test "returns false when seats limit is not exceeded", %{account: account} do
+      {:ok, account} =
+        Domain.Accounts.update_account(account, %{
+          limits: %{monthly_active_users_count: 10}
+        })
+
+      assert users_limit_exceeded?(account, 10) == false
+    end
+
+    test "returns true when seats limit is exceeded", %{account: account} do
+      {:ok, account} =
+        Domain.Accounts.update_account(account, %{
+          limits: %{users_count: 10}
+        })
+
+      assert users_limit_exceeded?(account, 11) == true
+    end
+
+    test "returns true when seats limit is not set", %{account: account} do
```

Comment:
Is this test named correctly?  Should it say `returns false when seats limit is not set` since you're checking against `== false`?

Reply:
You right, it's a typo in the name